### DISCUSSION TITLE
Disable printing feature details in the console/log

### DISCUSSION THREAD

**Discussion by jella-pradeep-kumar** [cucumber/common#2173] [Q&A Everything Else]
How to disable printing feature details in the console/log?

**Comment by mpkorstanje** [comment]
It depends which flavour of Cucumber are you using and how are you running it?

**Reply by jella-pradeep-kumar** [reply]
I am using cucumber with Test NG (JAVA)

**Reply by mpkorstanje** [reply]
If you're on a recent version, it will not print anything by default. This means that things are printed by the "pretty" plugin. Which you can remove from your configuration. If you are on an older version, it very much depends, and probably the quickest way to find out would be to look at the source code. Or upgrade to the latest.

**Comment by jella-pradeep-kumar** [comment]
Thanks, after removing pretty plug in, it was not printing the log
